docs/source/models/gbd2017/causes/tuberculosis/tuberculosis.rst

```diff
@@ -0,0 +1,164 @@
+.. _tuberculosis_model:
+
+=====================
+Modeling Tuberculosis
+=====================
+
+TB Overview
+-----------
+Tuberculosis (TB) is a disease caused by the bacteria Mycobacterium
+tuberculosis. It most commonly affects the lungs, although it can affect
+other parts of the body. TB can be fatal if not recognized and treated.
+It also can spread from person to person to infect others. Anyone who 
+inhales the airborne droplets that contains TB bacteria is called a "contact".
+A contact can be someone who living with the person who has infectious TB,
+like a family member, friend, or coworker. `UpToDate <https://www.uptodate.com/
+contents/tuberculosis-beyond-the-basics>`_
+
+What is Latent TB?
+------------------
+Latent tuberculosis infection (LTBI) is defined as a state of persistent
+immune response to stimulation by Mycobacterium tuberculosis antigens with
+no evidence of clinically manifest active TB. (WHO)
+
+What is Active TB?
+------------------
+Active tuberculosis, also known as reactivation TB. It occurs when individual's
+immune system becomes weakened and is no longer able to contain the latent bacteria;
+then the TB bacteria become "active", overwhelm the immune system and cause a person
+to become ill. (UpToDate) In GBD, Active TB consist of 6 causes: Drug-susceptible
+TB, Multidrug-resistant TB, Extensively drug-resistant TB, HIV/AIDS - Drug-susceptible
+TB, HIV/AIDS - Multidrug-resistant TB, and HIV/AIDS - Extensively drug-resistant TB.
+
+TB Progression
+--------------
+Active TB disease developed within 2 years called fast progression, Individuals
+who do not have rapid progression are classified as having slow-progressing latent
+tuberculosis infection. With latent infection, individuals experience no adverse
+health effects and will not transmit Mycobacterium tuberculosis, but they face
+an ongoing risk of developing active tuberculosis through reactivation.
+
+What is TB/HIV model?
+---------------------
+It's a complex joint model that includes both tuberculosis and HIV/AIDS in the model.
+We model TB and HIV together since GBD has also studied the impact of HIV on TB as well.
+Note that we have excluded the active TB drug resistance for our model and GBD predicted
+the proportion of new TB cases with MDR-TB or XDR-TB by ST-GPR.
+(GBD appendix: 03_Hmwe_TB_Nonfatal2017_resub)  
+
+GBD hierarchy
+-------------
+.. image:: cause_hierarchy.svg
+
+- {Anemia} = [no_anemia, mild_anemia, moderate_anemia, severe_anemia]
+- {HIV/AIDS} = [early_hiv, symptomatic_hiv, hiv_aids_with_art, aids]
+The causes in the GBD at any level of hierarchy are mutually exclusive
+and collectively exhaustive. 
+
+Cause model diagram
+------------------------
+.. image:: joint_model_diagram.svg

```

Comment:
I think the cause model diagram should have a key mapping the label in each state to what it stands for. I see that the data sources table below has definitions for each state, which is good, but I would also like to have a quick reference that tells me that, e.g. "LTBI, S_HIV" stands for "Latent Tuberculusis Infection, Susceptible to HIV" (is that accurate?).